<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Drop</class>
 <widget class="QDialog" name="Drop">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1061</width>
    <height>780</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QCheckBox" name="checkBox00">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>160</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox01">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>160</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox02">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>160</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox03">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>160</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox04">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>160</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox05">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>160</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox06">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>160</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox07">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>160</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox08">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>160</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox09">
   <property name="geometry">
    <rect>
     <x>970</x>
     <y>160</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonConfim">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>710</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Confim</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox17">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>330</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox11">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>330</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox16">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>330</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox15">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>330</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox19">
   <property name="geometry">
    <rect>
     <x>970</x>
     <y>330</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox13">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>330</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox10">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>330</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox18">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>330</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox14">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>330</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox12">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>330</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox27">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>500</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox21">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>500</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox26">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>500</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox25">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>500</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox29">
   <property name="geometry">
    <rect>
     <x>970</x>
     <y>500</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox23">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>500</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox20">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>500</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox28">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>500</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox24">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>500</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox22">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>500</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox37">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>670</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox31">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>670</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox36">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>670</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox35">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>670</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox39">
   <property name="geometry">
    <rect>
     <x>970</x>
     <y>670</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox33">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>670</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox30">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>670</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox38">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>670</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox34">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>670</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox32">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>670</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label00">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>90</width>
     <height>130</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label01">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>20</y>
     <width>90</width>
     <height>130</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label02">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>20</y>
     <width>90</width>
     <height>130</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label03">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>20</y>
     <width>90</width>
     <height>130</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label04">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>20</y>
     <width>90</width>
     <height>130</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label05">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>20</y>
     <width>90</width>
     <height>130</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label06">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>20</y>
     <width>90</width>
     <height>130</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label07">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>20</y>
     <width>90</width>
     <height>130</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label08">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>20</y>
     <width>90</width>
     <height>130</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label09">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>20</y>
     <width>90</width>
     <height>130</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label14">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>190</y>
     <width>90</width>
     <height>130</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label16">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>190</y>
     <width>90</width>
     <height>130</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label19">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>190</y>
     <width>90</width>
     <height>130</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label11">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>190</y>
     <width>90</width>
     <height>130</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label12">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>190</y>
     <width>90</width>
     <height>130</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label18">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>190</y>
     <width>90</width>
     <height>130</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label10">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>90</width>
     <height>130</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label15">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>190</y>
     <width>90</width>
     <height>130</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label13">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>190</y>
     <width>90</width>
     <height>130</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label17">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>190</y>
     <width>90</width>
     <height>130</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label24">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>360</y>
     <width>90</width>
     <height>130</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label26">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>360</y>
     <width>90</width>
     <height>130</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label29">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>360</y>
     <width>90</width>
     <height>130</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label21">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>360</y>
     <width>90</width>
     <height>130</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label22">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>360</y>
     <width>90</width>
     <height>130</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label28">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>360</y>
     <width>90</width>
     <height>130</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label20">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>360</y>
     <width>90</width>
     <height>130</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label25">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>360</y>
     <width>90</width>
     <height>130</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label23">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>360</y>
     <width>90</width>
     <height>130</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label27">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>360</y>
     <width>90</width>
     <height>130</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label34">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>530</y>
     <width>90</width>
     <height>130</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label36">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>530</y>
     <width>90</width>
     <height>130</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label39">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>530</y>
     <width>90</width>
     <height>130</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label31">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>530</y>
     <width>90</width>
     <height>130</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label32">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>530</y>
     <width>90</width>
     <height>130</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label38">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>530</y>
     <width>90</width>
     <height>130</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label30">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>530</y>
     <width>90</width>
     <height>130</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label35">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>530</y>
     <width>90</width>
     <height>130</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label33">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>530</y>
     <width>90</width>
     <height>130</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label37">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>530</y>
     <width>90</width>
     <height>130</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="background">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1061</width>
     <height>780</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>background</zorder>
  <zorder>checkBox00</zorder>
  <zorder>checkBox01</zorder>
  <zorder>checkBox02</zorder>
  <zorder>checkBox03</zorder>
  <zorder>checkBox04</zorder>
  <zorder>checkBox05</zorder>
  <zorder>checkBox06</zorder>
  <zorder>checkBox07</zorder>
  <zorder>checkBox08</zorder>
  <zorder>checkBox09</zorder>
  <zorder>pushButtonConfim</zorder>
  <zorder>checkBox17</zorder>
  <zorder>checkBox11</zorder>
  <zorder>checkBox16</zorder>
  <zorder>checkBox15</zorder>
  <zorder>checkBox19</zorder>
  <zorder>checkBox13</zorder>
  <zorder>checkBox10</zorder>
  <zorder>checkBox18</zorder>
  <zorder>checkBox14</zorder>
  <zorder>checkBox12</zorder>
  <zorder>checkBox27</zorder>
  <zorder>checkBox21</zorder>
  <zorder>checkBox26</zorder>
  <zorder>checkBox25</zorder>
  <zorder>checkBox29</zorder>
  <zorder>checkBox23</zorder>
  <zorder>checkBox20</zorder>
  <zorder>checkBox28</zorder>
  <zorder>checkBox24</zorder>
  <zorder>checkBox22</zorder>
  <zorder>checkBox37</zorder>
  <zorder>checkBox31</zorder>
  <zorder>checkBox36</zorder>
  <zorder>checkBox35</zorder>
  <zorder>checkBox39</zorder>
  <zorder>checkBox33</zorder>
  <zorder>checkBox30</zorder>
  <zorder>checkBox38</zorder>
  <zorder>checkBox34</zorder>
  <zorder>checkBox32</zorder>
  <zorder>label00</zorder>
  <zorder>label01</zorder>
  <zorder>label02</zorder>
  <zorder>label03</zorder>
  <zorder>label04</zorder>
  <zorder>label05</zorder>
  <zorder>label06</zorder>
  <zorder>label07</zorder>
  <zorder>label08</zorder>
  <zorder>label09</zorder>
  <zorder>label14</zorder>
  <zorder>label16</zorder>
  <zorder>label19</zorder>
  <zorder>label11</zorder>
  <zorder>label12</zorder>
  <zorder>label18</zorder>
  <zorder>label10</zorder>
  <zorder>label15</zorder>
  <zorder>label13</zorder>
  <zorder>label17</zorder>
  <zorder>label24</zorder>
  <zorder>label26</zorder>
  <zorder>label29</zorder>
  <zorder>label21</zorder>
  <zorder>label22</zorder>
  <zorder>label28</zorder>
  <zorder>label20</zorder>
  <zorder>label25</zorder>
  <zorder>label23</zorder>
  <zorder>label27</zorder>
  <zorder>label34</zorder>
  <zorder>label36</zorder>
  <zorder>label39</zorder>
  <zorder>label31</zorder>
  <zorder>label32</zorder>
  <zorder>label38</zorder>
  <zorder>label30</zorder>
  <zorder>label35</zorder>
  <zorder>label33</zorder>
  <zorder>label37</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
